src/helpers/pollUpdates.ts

```diff
@@ -53,7 +76,7 @@ export const poll = (store: Store) => async() => {
 };
 
 export default (store: Store): Cancel => {
-  if (APP_ENV !== 'production') {
+  if (APP_ENV == 'production') {
```

Comment:
Wonder if this guard can be reworked?  Locally I made a `public/rex/environment.json` file to have something to pull GA IDs from.  I had to disable this guard here so that it would be fetched.

Reply:
i've been developing all this `pollUpdates` stuff pretty much purely relying on unit tests, but amanda also requested that this somehow work in development so maybe we should figure that out. i'm not sure how to set it up in a super useful way though

Reply:
I changed it to guard specifically against the `test` env and when in development it checks for the `public/rex/environment.json` file and only proceeds if it is there.